### PR TITLE
pkg/cluster,controller,dashboard: Return -1 from attach client if exit code unknown

### PR DESCRIPTION
**Pull request by jvatic**

### CODE REVIEW

**Comment by jvatic** [comment]
ping @titanous

File: controller/jobs.go [hunk 1]
```diff
@@ -279,7 +279,12 @@ func (c *controllerAPI) JobLog(ctx context.Context, w http.ResponseWriter, req *
 			return
 		}
 		if tail {
-			ch <- &sseLogChunk{Event: "exit", Data: []byte(fmt.Sprintf(`{"status": %d}`, exit))}
+			if exit == -1 {
+				// Don't send exit status if we don't know it
+				ch <- &sseLogChunk{Event: "exit"}
```
Comment (titanous) [inline controller/jobs.go]: No need for this event, we already send eof

Comment (jvatic) [inline controller/jobs.go]: `exit` is sent instead of `eof` (see `return` below), should this change?

Comment (titanous) [inline controller/jobs.go]: Yeah, if we don't know the exit status, we should send eof

**Comment by titanous** [comment]
LGTM

**Pull request merged**